lib/kindle_highlights/client.rb

```diff
@@ -40,6 +42,8 @@ def highlights_for(asin)
     attr_accessor :email_address, :password, :mechanize_options
 
     def conditionally_sign_in_to_amazon
+      retries ||= 0
```

Comment:
Otherwise it'll cause an infinite loop when the `retry` takes effect. `||=` preserves the value from the past retry.